app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -305,18 +306,11 @@ class CatalogueController @Inject()(
             envName -> deployments.filter(_.environmentMapping.name == envName)
           }.toMap
 
-          /*
-           * Note that we ignore libraryDependencies obtained from parsing master / GitHub, and instead use those
-           * from the 'latest' slug.  As such, 'Platform Dependencies' on all tabs (including 'Latest') is populated
-           * from slug info.
-           * Slugs do not contain build related information however.  So 'Build Tools' continues to display information
-           * parsed from master / GitHub.
-           */
           Ok(
             serviceInfoPage(
               repositoryDetails.copy(environments = optDeployedEnvironments, jenkinsURL = jenkinsLink),
-              optMasterDependencies.map(_.copy(libraryDependencies = librariesOfLatestSlug)),
-              librariesBySlugVersion,
+              optMasterDependencies,
+              librariesBySlugVersion + (Latest.asString -> librariesOfLatestSlug),
```

Comment:
if you just import `SlugInfoFlag` above, we get a namespace to give context (`SlugInfoFlag.Latest`)

Reply:
Rationale as to why I wanted the focus to be on `Latest`, and why the fact it comes from `SlugInfoFlag` isn't actually of interest here below.